.github/workflows/lint_and_install.yml

```diff
@@ -0,0 +1,123 @@
+name: CG CI
+
+on: ["push", "pull_request"]
+
+jobs:
+  linting:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+    # Check out cg code
+    - uses: actions/checkout@v2
+
+    # Set up python
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+
+    - name: Install Lint Dependencies
+      run: |
+        pip install flake8 black
+
+    - name: Run linters
+      uses: samuelmeuli/lint-action@v1
+      # Let black change files
+      with:
+        github_token: ${{ secrets.github_token }}
+        auto_fix: true
+        # Enable linters
+        black: true
+        black_args: "--line-length 100"
+        # stop the build if there are Python syntax errors or undefined names
+        flake8: true
+        flake8_args: "cg/ --count --select=E9,F63,F7,F82 --show-source --statistics"
+
+
+  installations:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+
+    # Check out cg code
+    - name: Check out git repository
+      uses: actions/checkout@v2
+
+    # Set up python
+    - name: Set up Python ${{ matrix.python-version}}
+      uses: actions/setup-python@v2
+      # [0|0|436] [mans.magnusson@hasta:~] [P_main] 3s $ date
+      # Wed May 27 16:44:09 CEST 2020
+      # [0|0|436] [mans.magnusson@hasta:~] [P_main] $ python --version
+      # Python 3.7.1
+      # (prod170926) [hiseq.clinical@hippocampus ~]$ date
+      # Wed May 27 16:44:09 CEST 2020
+      # (prod170926) [hiseq.clinical@hippocampus ~]$ python --version
+      # Python 3.6.2 :: Continuum Analytics, Inc.
+      # (prod170926) [hiseq.clinical@hippocampus ~]$ pip --version
+      # pip 9.0.1
```

Comment:
Can we store this information somewhere instead of in comments here?

Reply:
I was also thinking about it but could not get any idea where... any suggestion? I think it is good to show why we are choosing these specific versions and where to update information when environments are updated

Reply:
Maybe a markdown file in `cg/.github/workflows`?

Reply:
Good idea! I added that